base/linalg/lapack.jl

```diff
@@ -657,6 +667,7 @@ for (gesvx, elty) in
         function gesvx!(fact::Char, trans::Char, A::StridedMatrix{$elty},
                AF::StridedMatrix{$elty}, ipiv::Vector{BlasInt}, equed::Char,
                R::Vector{$elty}, C::Vector{$elty}, B::StridedVecOrMat{$elty})
+            @chktrans
             lda, n    = size(A)
```

Comment:
This is not correct when `A` is a `SubArray`. It should be `lda=stride(A,2)`. The same is true for the calculations of the leading dimensions below.